package.json

```diff
@@ -37,93 +37,93 @@
   "dependencies": {
     "babel-polyfill": "^6.23.0",
     "base64-js": "^1.2.0",
-    "brace": "0.7.0",
-    "deep-extend": "0.4.1",
+    "brace": "0.10.0",
```

Comment:
yeah. Then I think we should be using webpack to control it, so that only one brace gets included on the page.